cmd/nerdctl/compose/compose_images.go

```diff
@@ -131,7 +133,7 @@ func printComposeImages(ctx context.Context, cmd *cobra.Command, containers []co
 				return err
 			}
 
-			size, err := unpackedImageSize(ctx, sn, image)
+			size, err := utils.UnpackedImageSize(ctx, sn, image)
```

Comment:
I feel `pkg/imgutil` might be a better place for this func since it doesn't depend on `cmd` related inputs (flags etc) and more related to `img`.

https://github.com/containerd/nerdctl/blob/main/pkg/imgutil/imgutil.go